.github/workflows/Bump Version.yml

```diff
@@ -0,0 +1,50 @@
+name: "Version Increment"
```

Comment:
Newer version of the bumb version. This will always bump on a new push.

Only works on branches that are not protected. Tested it, and correctly bumped the version on push, meaning we wont need to manually bump the version after already commiting the new changes.

Currently without a version bump, when ever someone pushes a change while forgetting to bump the version all of the changes are pushed to the old version, and then finally the new version leaving both for example versions 1.0.5 and 1.0.6 the exact same for a time.